docs/developers/nft-tokens.md

```diff
@@ -0,0 +1,400 @@
+This way, one can issue a semi fungible token or a non fungible token which is quite similar to an ESDT, but has a few more attributes, such as a changeable URI. 
+Once owning a quantity of a NFT/SFT, users will have their data store directly under their account, inside the trie. All the fields available inside a NFT/SFT token are:
+
+- attributes
+- balance
+- creator
+- hash
+- name
+- properties
+- royalties
+- tokenIdentifier
+- URIs
+
+**The flow of issuing and transferring non-fungible or semi-fungible tokens is:**
+- register/issue the token
+- set roles to the address that will create the NFT/SFTs
+- create the NFT/SFT
+- transfer quantity(es)
+
+## **Issuance of Non Fungible Tokens**
+
+One has to perform an issuance transaction in order to register the token. 
+Non Fungible Tokens are issued via a request to the Metachain, which is a transaction submitted by the Account which will manage the tokens. When issuing a token, one must provide a token name, a ticker and optionally additional properties. This transaction has the form:
+
+```
+IssuanceTransaction {
+    Sender: <account address of the token manager>
+    Receiver: erd1qqqqqqqqqqqqqqqpqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqzllls8a5w6u
+    Value: 5000000000000000000 (5 eGLD)
+    GasLimit: 60000000
+    Data: "issueNonFungible" +
+          "@" + <token name in hexadecimal encoding> +
+          "@" + <token ticker in hexadecimal encoding>
+}
+```
+
+Optionally, the properties can be set when issuing a token. Example:
+```
+IssuanceTransaction {
+    Sender: <account address of the token manager>
+    Receiver: erd1qqqqqqqqqqqqqqqpqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqzllls8a5w6u
+    Value: 5000000000000000000 (5 eGLD)
+    GasLimit: 60000000
+    Data: "issueNonFungible" +
+          "@" + <token name in hexadecimal encoding> +
+          "@" + <token ticker in hexadecimal encoding> +
+          "@" + <"canFreeze" hexadecimal encoded> + "@" + <"true" or "false" hexadecimal encoded> +
+          "@" + <"canWipe" hexadecimal encoded> + "@" + <"true" or "false" hexadecimal encoded> +
+          "@" + <"canPause" hexadecimal encoded> + "@" + <"true" or "false" hexadecimal encoded> +
+          "@" + <"canTransferNFTCreateRole" hexadecimal encoded> + "@" + <"true" or "false" hexadecimal encoded>
+}
+```
+
+The receiver address `erd1qqqqqqqqqqqqqqqpqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqzllls8a5w6u` is a built-in system smart contract (not a VM-executable contract), which only handles token issuance and other token management operations, and does not handle any transfers.
+The contract will add a random string to the ticker thus creating the **token identifier**. The random string starts with “-” and has 6 more random characters. For example, a token identifier could look like _ALC-6258d2_.
+
+## **Issuance of Semi Fungible Tokens**
+
+One has to perform an issuance transaction in order to register the token.
```

Comment:
This paragraph is almost the same as the one for the nonfungible tokens

Reply:
changed